modules/core/core-image/src/main/java/com/enonic/xp/core/impl/image/effect/ImageScales.java

```diff
@@ -0,0 +1,103 @@
+package com.enonic.xp.core.impl.image.effect;
+
+import com.google.common.base.Preconditions;
+
+import com.enonic.xp.core.impl.image.ImageFunction;
+import com.enonic.xp.core.impl.image.parser.CommandArgumentParser;
+import com.enonic.xp.image.FocalPoint;
+
+public class ImageScales
+{
+    private final int maxSideLength;
+
+    public ImageScales( final int maxSideLength )
+    {
+        this.maxSideLength = maxSideLength;
+    }
+
+    public ImageFunction full( FocalPoint focalPoint, Object... args )
+    {
+        Preconditions.checkArgument( args.length == 0, "Too many arguments %s", args.length );
```

Comment:
@rymsha error message can confused a user. Please update messages also for rest of the checks arguments.

Reply:
Message is
Too many arguments: 1

Only 0 arguments is allowed in this function